prow/plugins/cla/cla.go

```diff
@@ -278,10 +276,18 @@ func handleComment(gc gitHubClient, log *logrus.Entry, e *github.GenericCommentE
 					}
 				}
 			}
-
-			// No need to consider other contexts once you find the one you need.
-			break
```

Comment:
Since there's no break anymore, now we _are_ potentially adding/removing labels twice.

What I asked for in last review was to make the decision before acting, e.g.
```golang
claState := github.StatusPending
for _, status := range combined.Statuses {
  // code that sets claState
}
// code that sets labels based on claState
```